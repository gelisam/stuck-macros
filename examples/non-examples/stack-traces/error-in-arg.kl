#lang "prelude.kl"

(define fail (lambda (thing) (+ thing 1)))

(example (fail (error 'bad)))
