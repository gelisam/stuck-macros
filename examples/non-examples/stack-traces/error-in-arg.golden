Expected 3 entries between parentheses, but got
  #[error-in-arg.kl:3.28-3.45]
   <(#%app + (error msg) 1)>
